frontend/src/components/SignUpModal.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import {
+    Button,
+    AbsoluteCenter,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    // ModalCloseButton,
+    SimpleGrid,
+    List,
+    ListItem,
+    ListIcon,
+    Box,
+    Text,
+  } from '@chakra-ui/react'
+import { ButtonPrimary } from "./ButtonPrimary";
+import { MdIosShare } from "react-icons/md";
+import { MdAutoAwesome } from "react-icons/md";
+import { MdHistory } from "react-icons/md";
+import { MdHandshake } from "react-icons/md";
+
+
+const SignUpModal = ({ closeModal, modalOpen }) => {
+    const navigate = useNavigate();
+
+    const routeSignUp = () => {
+        navigate("/signup");
+    }
+
+    return (
+        <Modal size={"2xl"} isOpen={modalOpen} onClose={() => closeModal(false)}>
```

Comment:
please add `autoFocus={false}` to disable auto focusing on modal. i typed too quickly the first time i tried to export and it went through the modal too quick. this should fix that experience